packages/wp-dashboard/src/api/pages.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { addQueryArgs } from '@web-stories-wp/design-system';
+
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Search pages by search term.
+ * Used for "Stories Archive" custom page selection on settings page.
+ *
+ * @param {string} searchTerm Search term.
+ * @param {string} apiPath API path.
+ * @return {Promise} Request promise.
+ */
+export function searchPages(searchTerm, apiPath) {
+  const path = addQueryArgs(apiPath, {
+    per_page: 100,
+    search: searchTerm,
+    _fields: ['id', 'title'],
```

Comment:
These fields could be a const.

Reply:
https://github.com/google/web-stories-wp/pull/9237/commits/fe81622d0a1ab8dd66dab337f0e8f2ee43afb20a